sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -119,42 +128,44 @@ Once completed, set the values of the client ID, tenant ID, and client secret of
 
 ```python
 from azure.identity import DefaultAzureCredential
-from azure.ai.formrecognizer import FormRecognizerClient
+from azure.ai.formrecognizer import DocumentAnalysisClient
 credential = DefaultAzureCredential()
 
-form_recognizer_client = FormRecognizerClient(
+document_analysis_client = DocumentAnalysisClient(
     endpoint="https://<my-custom-subdomain>.cognitiveservices.azure.com/",
     credential=credential
 )
 ```
 
 ## Key concepts
 
-### FormRecognizerClient
-`FormRecognizerClient` provides operations for:
+### DocumentAnalysisClient
+`DocumentAnalysisClient` provides operations for analyzing input documents using custom and prebuilt models through the `begin_analyze_document` and `begin_analyze_document_from_url` APIs.
+Use the `model_id` parameter to select the type of model for analysis.
 
- - Recognizing form fields and content using custom models trained to recognize your custom forms. These values are returned in a collection of `RecognizedForm` objects.
- - Recognizing common fields from the following form types using prebuilt models. These fields and metadata are returned in a collection of `RecognizedForm` objects.
-    - Sales receipts. See fields found on a receipt [here][service_recognize_receipt].
-    - Business cards. See fields found on a business card [here][service_recognize_business_cards].
-    - Invoices. See fields found on an invoice [here][service_recognize_invoice].
-    - Identity documents. See fields found on identity documents [here][service_recognize_identity_documents].
- - Recognizing form content, including tables, lines, words, and selection marks, without the need to train a model. Form content is returned in a collection of `FormPage` objects.
+|Model ID| Features
+|-|-
+|"{custom-model-id}" | Text extraction, selection marks, tables, labeled fields and values from your custom documents
+|"prebuilt-layout" | Text extraction, selection marks, tables
+|"prebuilt-document" | Text extraction, selection marks, tables, key-value pairs and entities
+|"prebuilt-invoices" | Text extraction, selection marks, tables, and pre-trained fields and values pertaining to invoices
+|"prebuilt-businessCard" | Text extraction and pre-trained fields and values pertaining to business cards
+|"prebuilt-idDocument" | Text extraction and pre-trained fields and values pertaining to identity documents
+|"prebuilt-receipt" | Text extraction and pre-trained fields and values pertaining to US sales receipts
 
-Sample code snippets are provided to illustrate using a FormRecognizerClient [here](#recognize-forms-using-a-custom-model "Recognize Forms Using a Custom Model").
+Sample code snippets are provided to illustrate using a DocumentAnalysisClient [here](#analyze-documents-using-a-custom-model "Analyze Documents Using a Custom Model").
```

Comment:
Just wondering whether this link should point to the general samples folder? The way it is worded, made me thing that that's what was going to be linked. Or if not, maybe reword to say something like:
```suggestion
Sample code snippets are provided to illustrate using a DocumentAnalysisClient with a custom model [here](#analyze-documents-using-a-custom-model "Analyze Documents Using a Custom Model").
```

Reply:
this was added because in ux studies people wouldn't continue to scroll down on the readme and see the examples. I'm going to update both the DAC and DMAC to link to the examples header.